build.xml

```diff
@@ -44,7 +44,7 @@
   <property name="spotbugs.home" value="${ivy.dir}/spotbugs-${spotbugs.version}" />
   <property name="spotbugs.jar" value="${spotbugs.home}/lib/spotbugs-ant.jar" />
 
-  <property name="apache-rat.version" value="0.16.1" />
+  <property name="apache-rat.version" value="0.15" />
```

Comment:
Please sync with `master` branch. This regression is tangential to NUTCH-3032. Thanks